tests/functional/authentication/01-remote-suite-same-name.t

```diff
@@ -34,7 +34,7 @@ scp ${SSH_OPTS} -pqr "${TEST_SOURCE_DIR}/${TEST_NAME_BASE}/"* \
 # shellcheck disable=SC2086
 run_ok "${TEST_NAME_BASE}-register" \
     ssh ${SSH_OPTS} "${CYLC_TEST_HOST}" \
-    CYLC_VERSION="$(cylc version)" cylc register "${SUITE_NAME}" \
+    CYLC_VERSION="$(cylc version --short)" cylc register "${SUITE_NAME}" \
```

Comment:
In Cylc7 we had `cylc version` and `cylc version --long`, could we swap `--short` for `--long`, It seems safer to only give extra when the user requests extra.

Reply:
I considered that and came down on the other side (in the description):
> I considered requiring --long to get the location, but people may not be expecting additional options to a version query, (Also, `rose version` prints the location).

To expand on this a bit, on a system that may have multiple cylc-flow versions installed in both conda and pip virtual environments, and where the short version is no longer fine grained (via `git describe`), the installed path is more definitive than the release version string.  

Also: as I noted, `rose version` prints the path - are you going to change that?  And, what are your use cases that *require* the short format?  Presumably you mean just reporting the version string for `conda install cylc-flow=8.0a3` etc.?  IMO `cylc version` long format doesn't hinder that as no one is going it think that the local installation path is part of that version string.

Reply:
(This isn't a big deal, obviously, but I given the reasons above I thought it would be good to show which cylc is in use by default, so long as that doesn't cause other problems).

Reply:
I think showing version with `cylc version` by default, and showing extra info if required makes more sense (least surprise). But either way is fine for me.

Reply:
Keeping my +1 here. Now up to @oliver-sanders the final review/merge :+1:

Reply:
> it's a little surprising to see a member of the Rose team disagree rofl)

Not at all, I'm very glad for the chance to re-engineer Rose, a lot of really nasty behaviour there.